test/ts-test.ts

```diff
@@ -0,0 +1,98 @@
+
```

Comment:
Thank you for writing tests! I really appreciate it.

This file is not using mocha, though, so it can be confusing. Can you try to make it a real test? This article can help: https://journal.artfuldev.com/unit-testing-node-applications-with-typescript-using-mocha-and-chai-384ef05f32b2

Or, if you want to just keep it as an example of how typescript would work, then can you rename this file to `typescript-example.ts`, so it doesn't look like a test.

Reply:
done. One can run the tests with `mocha -r ts-node/register test/ts-test.ts`.